src/Components/ListArticles.jsx

```diff
@@ -0,0 +1,52 @@
+import {useEffect, useState} from "react";
+import axios from "axios";
+import {Button, Card, ListGroup} from "react-bootstrap";
+
+export const ListArticles = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    const getArticles = async () => {
+      try {
+        const reqLink = "http://ec2-35-179-90-244.eu-west-2.compute.amazonaws.com/api/articles";
+        const response = await axios.get(reqLink);
+        const data = response.data;
+        setArticles(data.articles);
+      } catch (error) {
+        console.log(error);
+      }
+
+    }
+
+    getArticles();

```

Comment:
I'd recommend extracting this function to an api utils file will help keep things tidy as you go on